pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,60 @@
+{stdenv, python37, python37Packages, gettext, qt5, fetchFromGitHub}:
```

Comment:
There's no need to import both `python37` and `python37Packages`. Use here just `python` or `pythonPackages` and when you `callPackage`, use `python = `python3;`. Also, use just the `3` suffix in `callPackage` so it will use the latest and greatest Python version as time goes on.

Reply:
@doronbehar 
>and when you callPackage

`callPackage` in `pkgs/top-level/all-packages.nix` ?